### ISSUE TITLE
[Az.Functions] Need a cmdlet to install the Core Tools

### ISSUE THREAD

**Issue by Francisco-Gamino** [Azure/azure-functions-core-tools#2014]
## Description
In order to do local development for functions, the customer needs the Az.Functions module + Core Tools. Need a cmdlet that installs the Core Tools.

Make sure we have a way to upgrade to the next version.

**Comment by Francisco-Gamino** [comment]
cc: @anirudhgarg @eamonoreilly @anthonychu

**Comment by anthonychu** [comment]
Makes sense. Should this script be hosted on the CDN as well (to be consumed by VS Code extension, choco package, etc)?

**Comment by Francisco-Gamino** [comment]
I was thinking to host the script in the Core-Tools repo and have a `https://aka.ms` link point to it.  The PowerShell team does something like this. Here is how the script will get executed  on the customer side from PowerShell: 

`iex "& { $(irm 'https://aka.ms/install-powershell.ps1')} -UseMSI"`

where `https://aka.ms/install-powershell.ps1` points to `https://raw.githubusercontent.com/PowerShell/PowerShell/master/tools/install-powershell.ps1`

In our case, it will be `https://aka.ms/install-core-tools.ps1`

**Comment by anthonychu** [comment]
That sounds good to me. Will the script be signed?

**Comment by Francisco-Gamino** [comment]
This is going to be a thin wrapper for the MSI, so signing will not be required.
